src/cloudWatchLogs/registry/logDataRegistry.ts

```diff
@@ -336,41 +364,6 @@ export async function filterLogEventsFromUriComponents(
     }
 }
 
-export async function getLogEventsFromUriComponents(
```

Comment:
We can remove this whole function in favor of `filterLogEventsFromUriComponents` since they do the same thing, except one filters. So in the scenario we want everything we just dont pass in a filter.

Reply:
nice! I wondered about that...